index.js

```diff
@@ -70,6 +73,11 @@ function createServer(argv) {
     var ldp = ldpApp.locals.ldp;
     app.use(ldp.mount, ldpApp);
 
+    //Error handling
+    app.use(errorHandler.handler);
```

Comment:
I tried to mount it to the router first but it didn't work. I think it was because the handlers were calling res.sent instead of next(err) and the responses were being intercepted. I will attach it to the router again to check if it works now.